scripts/apps/authoring/authoring/directives/ArticleEditDirective.ts

```diff
@@ -324,9 +324,13 @@ export function ArticleEditDirective(
                             // draftjs editor state will be
                             // outdated after editing in modal
                             resetFieldMetadata(scope.item);
-                            scope.refresh();
 
-                            if (isPublished(scope.item)) {
+                            // On multiedit mode, there is no refresh function
+                            if (typeof scope.refresh === 'function') {
+                                scope.refresh();
+                            }
+
+                            if (isPublished(scope.item) || scope.articleEdit) {
```

Comment:
What are you checking with ` scope.articleEdit`? I don't see such a variable in the directive.

Reply:
If scope.articleEdit is defined then it means it's called from Multiedit directive, which doesn't have item and originItem in scope, so autosave is not working.

Reply:
Makes sense. Can you add TypeScript types for `scope` and add a comment in the interface that `articleEdit` means that the item is being edited in multi-edit mode?